src/views/blocks/rsvp/form.php

```diff
@@ -8,30 +8,34 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @since 4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   4.9
+ * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   TBD Add vars to docblock.
  *
  * @version 4.12.0
  *
+ * @var Tribe__Tickets__Editor__Template $template
```

Comment:
Isn't `$template` the `$this`?

Reply:
I was going to ask about this. Any template call should have `$this` available as reference to `Tribe__Tickets__Editor__Template`.

Reply:
fixed

Reply:
done